frontend/src/metabase/query_builder/containers/QueryBuilder.unit.spec.tsx

```diff
@@ -229,6 +260,43 @@ describe("QueryBuilder", () => {
 
         expect(screen.getByDisplayValue(TEST_CARD.name)).toBeInTheDocument();
       });
+
+      it("render time-series grouping widget for date field breakout", async () => {
+        await setup({
+          card: TEST_TIME_SERIES_WITH_DATE_BREAKOUT_CARD,
+        });
+        const timeSeriesModeFooter = await screen.findByTestId(
```

Comment:
nit:
- I'd call a "footer" the thing below this bar - [see screenshot](https://github.com/metabase/metabase/assets/6830683/cdadd931-a663-4e99-a81c-2e48b270f46e) ("footer" is something that is last, and there's nothing below it)
- ideally we match the `data-testid` name

```suggestion
        const timeSeriesModeBar = await screen.findByTestId(
```

Reply:
I was following [the component name](https://github.com/metabase/metabase/blob/master/frontend/src/metabase/visualizations/click-actions/components/TimeseriesModeFooter.tsx), not the test ID. I'll make sure they are the same. The thing is that we've been using `ModeFooter` [for a long time](https://github.com/metabase/metabase/blob/d3ff5fdc5fca87d3f7e01a39c8890d599a927260/frontend/src/metabase/query_builder/components/view/View.jsx#L112) (At least 4 years), so I'll stick to that for now.

Reply:
Fixed in 882f7b9f05435386dc3e5012817d9f00a3a78351